Index: GenerateOntology.mwe2
===================================================================
--- GenerateOntology.mwe2	(revision 339)
+++ GenerateOntology.mwe2	(working copy)
@@ -43,6 +43,7 @@
 
 			// generates Java API for the generated EPackages 
 			fragment = ecore.EcoreGeneratorFragment {
+				generateEdit = true
 			// referencedGenModels = "uri to genmodel, uri to next genmodel"
 			}
 
